### DISCUSSION TITLE
Switch docker container to rtorrent stickz?

### DISCUSSION THREAD

**Discussion by stickz** [crazy-max/docker-rtorrent-rutorrent#337] [General]
@crazy-max I have made a new version of libtorrent and rTorrent. It is based off libtorrent 0.13.8 and rTorrent 0.9.8. Many improvements from jesec are backported and all the patches here are supported. https://github.com/stickz/rtorrent/releases

How would you feel about switching the docker container to it? I will be actively tagging releases for stability. It currently functions much better than the patches. There are significant stability improvements. It's a drop in release with no changes required for compatibility.

**Comment by crazy-max** [comment]
Sure SGTM

**Reply by stickz** [reply]
I will add it next week.
